src/main/java/frc/team4909/robot/Robot.java

```diff
@@ -155,13 +162,30 @@ public void robotInit() {
     // manipulatorGamepad.buttonPressed(BionicF310.X, new ToggleCamera());
 
     /* Arm Setpoints */
-    manipulatorGamepad.buttonPressed(BionicF310.A, new CargoIntakeOnly());
-    manipulatorGamepad.buttonPressed(BionicF310.B, new HatchOnly());
-    manipulatorGamepad.buttonPressed(BionicF310.Y, new CargoOutOnly());
+    manipulatorGamepad.buttonPressed(BionicF310.A, new Arm135()); // tune first
+    manipulatorGamepad.buttonPressed(BionicF310.B, new Arm90()); // tune first
+    manipulatorGamepad.buttonPressed(BionicF310.Y, new Arm45()); // tune first
+
+    /* User Interface Setpoints */
+
+    // all of the following setpoints set both the elevator and the elevator arm
+    SmartDashboard.putData("Cargo High", new CargoHigh()); // top cargo spot
+    SmartDashboard.putData("Cargo Low", new CargoLow()); // bottom cargo spot
+    SmartDashboard.putData("Cargo Middle", new CargoMiddle()); // middle cargo soit
+    SmartDashboard.putData("Cargo Ship", new CargoShip()); // cargo spot for ship
+    SmartDashboard.putData("Hatch High", new HatchHigh()); // top hatch spot
+    SmartDashboard.putData("Hitch Low", new HatchLow()); // bottom hatch spot
+    SmartDashboard.putData("Hatch Middle", new HatchMiddle()); // middle hatch spot
+
+    
+    // SmartDashboard.putData(new ZeroElevator());
+    // SmartDashboard.putData(new ZeroStilts());
+
+
+
+
 
 
-    SmartDashboard.putData(new ZeroElevator());
-    SmartDashboard.putData(new ZeroStilts());
```

Comment:
can we leave the button there but hide it so we have it if necessary?

Reply:
I would argue that the button need not be hidden.